docs/README.md

```diff
@@ -0,0 +1,9 @@
+# EmailOTP Authenticator
+ 
+The Email OTP authenticator allows you to authenticate users via email services using Email APIs through WSO2 IS. 
+ 
```

Comment:
The Email OTP authenticator allows you to authenticate users via email services using email APIs through WSO2 Identity Server.

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/30/commits/0013e9ee1bd5e1a1d1d4f5970d968e1ed2c0be91